plugin/src/main/java/org/opensearch/ml/indices/MLIndicesHandler.java

```diff
@@ -100,7 +100,7 @@ public void initMLIndexIfAbsent(String indexName, String mapping, ActionListener
                     log.error("Failed to create index " + indexName, e);
                     listener.onFailure(e);
                 });
-                CreateIndexRequest request = new CreateIndexRequest(indexName);
+                CreateIndexRequest request = new CreateIndexRequest(indexName).mapping(mapping);
```

Comment:
Can we add a unit test for this? It can be added later if this needs to merge soon in order to catch 2.0

Reply:
yes, this needs to merge ASAP. Will add more test later